projects/aca-folder-rules/src/lib/rules-list/rule/rule-list-item.ui-component.ts

```diff
@@ -34,4 +35,18 @@ import { Rule } from '../../model/rule.model';
 export class RuleListItemUiComponent {
   @Input() rule: Rule;
   @Input() isSelected: boolean;
+  @Input() nodeId: string;
+
+  constructor(private folderRulesService: FolderRulesService) {}
+
+  onToggleClick() {
+    this.folderRulesService.toggleRule(this.nodeId, this.rule.id, this.parseRule(this.rule));
+  }
+
+  private parseRule(rule): Rule {
+    const parsedRule = { ...rule, enabled: !this.rule.enabled };
+    delete parsedRule.actions[0].params.actionContext;
+    parsedRule.conditions.booleanMode = parsedRule.conditions.booleanMode.toUpperCase();
```

Comment:
Why do we need to do this here?

Reply:
- we need delete `actionContext`  as it is read-only property, if we include it to the request, we receive an error.
- server receives only uppercase 'AND / OR' (bug)

However, those two seems to be solved. Tom's PR about the first point is already open, and Kristian fixed the second issue, but the env is not updated yet.